src/bin/huak/commands/add.rs

```diff
@@ -15,7 +16,7 @@ pub fn cmd() -> Command<'static> {
 
 /// Run the `add` subcommand.
 pub fn run(args: &ArgMatches) -> CliResult {
-    let _ = args.get_one::<String>("dependency");
-
-    unimplemented!()
+    let dependency = args.get_one::<String>("dependency");
+    add_project_dependency(dependency.unwrap().to_string(), false)?;
```

Comment:
```suggestion
    let dependency = args.get_one::<String>("dependency");
    
    let dependency = match args.get_one::<String>("dependency") {
        Some(d) => d,
        None => {
            return Err(CliError::new(HuakError::MissingArguments))
        }
    };
    
    ops::add::add_project_dependency(dependency, false)?;
```
I'm like 60% sure i got this write. I'll come back.

Reply:
Only change I think is adding `, 1)` to just return a basic exit code. A rebase will add status codes to CliError.